ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/SearchForWebmap/README.md

```diff
@@ -0,0 +1,22 @@
+#Search for webmap by keyword
+
+This sample demonstrates how find portal items by using a keyword, and limit the results to items of a certain type - in this case, webmaps.
+
+![](screenshot.png)
+
+##How it works
+1. A `Portal` is created up with a `Credential` using `OAuthModeUser`.
+2. When the app starts, the portal is loaded and the user is prompted to login.
+3. A search bar allows the user to enter a keyword (or tag) to search for - e.g. "usa". NOTE if multiple item types are required these can be set via the `types` property.
```

Comment:
Are keywords, tags, and item types all the same in this context? I think keywords and tags mean the same thing. If so, say `...enter a keyword (a tag) to ...`